src/current/releases/index.md

```diff
@@ -400,7 +418,7 @@ macOS downloads are **experimental**. Experimental downloads are not yet qualifi
 </section>
 
 <section class="filter-content" markdown="1" data-scope="windows">
-    Windows 8 or higher is required. Windows downloads are **experimental**. Experimental downloads are not yet qualified for production use and not eligible for support or uptime SLA commitments, whether they are for testing releases or production releases.
+    Windows 8 or higher is required. Windows downloads are **experimental**. Experimental downloads are not yet qualified for production use and not eligible for support or uptime SLA commitments, whether they are for testing releases or production releases
```

Comment:
Double-checking: this period was removed because it was generating double periods, right? In preview, it still shows one period at the end (which I think is correct).